package.json

```diff
@@ -18,7 +18,8 @@
     "prepare": "lerna bootstrap",
     "prepublishOnly": "yarn build",
     "reinstall": "echo Clearing yarn cache; yarn cache clean; echo Removing yarn offline mirror; rimraf .yarn-offline-mirror; echo Removing package folders; lerna clean --yes; rimraf node_modules; yarn install",
-    "start": "yarn info @carbon/vue --json > package-info.jsonl; vue-cli-service storybook:serve -p 9001 -c storybook/.storybook",
+    "start:a": "yarn info @carbon/vue --json > package-info.jsonl;",
```

Comment:
Why have the start lines been split?

Reply:
I had an issue running the original start command on Windows. found the solution in your comment on #910